pact/concrete-policies/non-fungible-policy/non-fungible-policy-v1.repl

```diff
@@ -28,6 +28,22 @@
   (marmalade-v2.abc.create-account "k:creator" (read-keyset 'creator-guard))
 (commit-tx)
 
+(begin-tx "Create a non-fungible token fails when precision is not 0")
+  (use marmalade-v2.ledger)
+  (use marmalade-v2.util-v1)
+
+  (env-data {
+    "token-id": (create-token-id { 'uri: "test-non-fungible-uri", 'precision: 1, 'policies: (create-policies util.DEFAULT_NON_FUNGIBLE_CONCRETE_POLICY) } )
+    ,"account": "k:account"
+    ,"nfp-mint-guard": {"keys": ["account"], "pred": "keys-all"}
```

Comment:
Can we remove `nfp-mint-guard` , as it's outdated. We've delegated the guarding to `guard-policy-v1` instead of policy-mint-guards.